frontend/packages/dev-console/src/components/gitops/GitOpsListPage.scss

```diff
@@ -0,0 +1,10 @@
+.odc-gitops-list-page-heading {
+    align-items: center;
+    display: inline-block;
+    font-weight: normal;
+    font-size: 12px;
+
+    &__argocd {
+        margin-right: var(--pf-global--spacer--sm);
+    }
```

Comment:
You don't need to specifically add margin like this. You can use `hasGutter` prop on `Split` component.